### PR TITLE
[zh-cn]: Update translation of Promise.any()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13733

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 1]
```diff
@@ -5,53 +5,45 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
 
 {{JSRef}}
 
-`Promise.any()` 接收一个由 {{JSxRef("Promise")}} 所组成的可迭代对象，该方法会返回一个新的 `promise`，一旦可迭代对象内的任意一个 `promise` 变成了兑现状态，那么由该方法所返回的 `promise` 就会变成兑现状态，并且它的兑现值就是可迭代对象内的首先兑现的 `promise` 的兑现值。如果可迭代对象内的 `promise` 最终都没有兑现（即所有 `promise` 都被拒绝了），那么该方法所返回的 `promise` 就会变成拒绝状态，并且它的拒因会是一个 {{JSxRef("Global_Objects/AggregateError", "AggregateError")}} 实例，这是 {{JSxRef("Error")}} 的子类，用于把单一的错误集合在一起。
+**`Promise.any()`** 静态方法将一个 Promise 可迭代对象作为输入，并返回一个 {{jsxref("Promise")}}。当输入的 Promise 中的任何一个兑现时，这个返回的 Promise 将会兑现，并返回第一个兑现的值。如果输入的所有 Promise 都被拒绝（包括传递的可迭代对象为空时），则返回的 Promise 将被拒绝，带有一个包含拒绝原因数组的 {{jsxref("AggregateError")}}。
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:8]:
```suggestion
**`Promise.any()`** 静态方法将一个 Promise 可迭代对象作为输入，并返回一个 {{jsxref("Promise")}}。当输入的任何一个 Promise 兑现时，这个返回的 Promise 将会兑现，并返回第一个兑现的值。当所有输入 Promise 都被拒绝（包括传递了空的可迭代对象）时，它会以一个包含拒绝原因数组的 {{jsxref("AggregateError")}} 被拒绝。
```

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 2]
```diff
@@ -5,53 +5,45 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
...
+同时，与 {{JSxRef("Promise.race()")}} 返回第一个*敲定*（无论是兑现还是拒绝）的值不同的是，该方法返回第一个*兑现*的值。该方法忽略所有被拒绝的 Promise，直到第一个被兑现的 Promise。
 
 ## 示例
 
-### First to fulfil
+### 使用 Promise.any()
 
 如果可迭代数组内的任意一个 `promise` 兑现了，那么该方法所返回的 `promise` 也会切换至兑现状态，哪怕首个敲定的 `promise` 是被拒的。不同的是，{{jsxref("Promise.race()")}} 所返回的 `promise` 的状态会跟随首个敲定的 `promise` 的状态。
 
+`Promise.any()` 会以第一个兑现的 Promise 而兑现，即使有 Promise 先被拒绝。这与 {{jsxref("Promise.race()")}} 不同，后者会使用第一个敲定的 Promise 来兑现或拒绝。
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:45]:
```suggestion
`Promise.any()` 会以第一个兑现的 Promise 来兑现，即使有 Promise 先被拒绝。这与 {{jsxref("Promise.race()")}} 不同，后者会使用第一个敲定的 Promise 来兑现或拒绝。
```
```suggestion
`Promise.any()` 会以第一个兑现的 Promise 而兑现，即使有 Promise 先被拒绝。这与 {{jsxref("Promise.race()")}} 不同，后者会使用第一个敲定的 Promise 来兑现或拒绝。
```

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 3]
```diff
@@ -5,53 +5,45 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
...
-
-- 如果传入了一个空的可迭代数组，那么该方法就会返回一个已经被拒 `promise`，其拒因是一个 `AggregateError` 实例，该实例的 `errors` 属性会是一个空数组。
+同时，与 {{JSxRef("Promise.race()")}} 返回第一个*敲定*（无论是兑现还是拒绝）的值不同的是，该方法返回第一个*兑现*的值。该方法忽略所有被拒绝的 Promise，直到第一个被兑现的 Promise。
 
 ## 示例
 
-### First to fulfil
+### 使用 Promise.any()
 
 如果可迭代数组内的任意一个 `promise` 兑现了，那么该方法所返回的 `promise` 也会切换至兑现状态，哪怕首个敲定的 `promise` 是被拒的。不同的是，{{jsxref("Promise.race()")}} 所返回的 `promise` 的状态会跟随首个敲定的 `promise` 的状态。
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:43]:
```suggestion

```

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 4]
```diff
@@ -5,53 +5,45 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
...
 ```
 
 ### 参数
 
 - `iterable`
-  - : 一个[可迭代](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol)的对象，例如 [Array](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array)。
+  - : 一个[可迭代对象](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)，例如一个 promise {{jsxref("Array", "数组", "", 1)}}。
 
-### 返回值
+### 返回值s
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:23]:
```suggestion
### 返回值
```

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 5]
```diff
@@ -5,52 +5,42 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
 
 {{JSRef}}
 
-`Promise.any()` 接收一个由 {{JSxRef("Promise")}} 所组成的可迭代对象，该方法会返回一个新的 `promise`，一旦可迭代对象内的任意一个 `promise` 变成了兑现状态，那么由该方法所返回的 `promise` 就会变成兑现状态，并且它的兑现值就是可迭代对象内的首先兑现的 `promise` 的兑现值。如果可迭代对象内的 `promise` 最终都没有兑现（即所有 `promise` 都被拒绝了），那么该方法所返回的 `promise` 就会变成拒绝状态，并且它的拒因会是一个 {{JSxRef("Global_Objects/AggregateError", "AggregateError")}} 实例，这是 {{JSxRef("Error")}} 的子类，用于把单一的错误集合在一起。
+**`Promise.any()`** 静态方法将一个 Promise 可迭代对象作为输入，并返回一个 {{jsxref("Promise")}}。当输入的任何一个 Promise 兑现时，这个返回的 Promise 将会兑现，并返回第一个兑现的值。当所有输入 Promise 都被拒绝（包括传递了空的可迭代对象）时，它会以一个包含拒绝原因数组的 {{jsxref("AggregateError")}} 被拒绝。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:8]:
```suggestion
**`Promise.any()`** 静态方法将一个 Promise 可迭代对象作为输入，并返回一个 {{jsxref("Promise")}}。当输入的任何一个 Promise 兑现时，这个返回的 Promise 将会兑现，并返回第一个兑现的值。当所有输入 Promise 都被拒绝（包括传递了空的可迭代对象）时，它会以一个包含拒绝原因数组的 {{jsxref("AggregateError")}} 拒绝。
```

File: files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md [hunk 6]
```diff
@@ -5,52 +5,42 @@ slug: Web/JavaScript/Reference/Global_Objects/Promise/any
...
-Promise.any(iterable);
+```js-nolint
+Promise.any(iterable)
 ```
 
 ### 参数
 
 - `iterable`
-  - : 一个[可迭代](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol)的对象，例如 [Array](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array)。
+  - : 一个[可迭代对象](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)，例如一个 promise {{jsxref("Array", "数组", "", 1)}}。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/promise/any/index.md:21]:
```suggestion
  - : 一个 promise 的[可迭代对象](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)（例如一个  {{jsxref("Array", "数组", "", 1)}}）。
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**